core-rust/state-manager/src/receipt.rs

```diff
@@ -136,6 +137,7 @@ pub struct LedgerTransactionReceipt {
 /// It is not verifiable against ledger, but may still be useful for debugging.
 #[derive(Debug, Clone, ScryptoCategorize, ScryptoEncode, ScryptoDecode)]
 pub struct LocalTransactionExecution {
+    pub execution_metrics: ExecutionMetrics,
```

Comment:
Very minor, but in terms of field ordering, I'd suggest it's less important than, and therefore should be ordered below `outcome`